thirdparty/github.go

```diff
@@ -1135,3 +1135,22 @@ func MergePullRequest(ctx context.Context, token, owner, repo, commitMessage str
 	}
 	return nil
 }
+
+func PostCommentToPullRequest(ctx context.Context, token, owner, repo, comment string, prNum int) error {
```

Comment:
As personal preference I like keeping like types together since it keeps function signatures clean but I don't feel super strongly about it either

Reply:
Yeah, I don't have strong feelings either way on this, I just feel like it might be more ergonomic from the perspective of a dev calling this function to send related arguments in an ordered group (since we always reference PRs by owner/repo/PR num) and send the last argument as the actual payload.